deploy/kubernetes/delete-driver.sh

```diff
@@ -9,14 +9,10 @@
 set -o nounset
 set -o errexit
 
+readonly DEPLOY_VERSION="${GCE_PD_DRIVER_VERSION:-stable}"
```

Comment:
can you update the args documentation in the header comment, what are the other options for this variable